pkg/plugins/golang/v3/scaffolds/internal/templates/api/webhook_suitetest.go

```diff
@@ -198,7 +189,8 @@ var _ = BeforeSuite(func() {
 	err = AddToScheme(scheme)
 	Expect(err).NotTo(HaveOccurred())
 
-	%s
+	err = admission%s.AddToScheme(scheme)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
The changes here do not seem required for that.
We have the marker to do it. See that it was removed here https://github.com/kubernetes-sigs/kubebuilder/pull/2619/files#r850232507